src/components/popover/popover.component.ts

```diff
@@ -93,4 +142,28 @@ export class PopoverComponent implements OnInit {
     selectTab(tab: TabName): void {
         this.activeTab = tab;
     }
+
+    onOverlay() {
+        this.cancel.emit(undefined);
+    }
+
+    private calculatePosition() {
+        const offsetLeft: number = this.wrapPosition.left + 265;
+        const popover: HTMLElement = this.popoverHost.nativeElement;
+        let newLeft: number = this.keyPosition.left + (this.keyPosition.width / 2);
+
+        this.leftArrow = newLeft <  offsetLeft;
+        this.rightArrow = (newLeft + popover.offsetWidth) > offsetLeft + this.wrapPosition.width;
+
+        if (this.leftArrow) {
+            newLeft = this.keyPosition.left;
+        } else if (this.rightArrow) {
+            newLeft = this.keyPosition.left - popover.offsetWidth + this.keyPosition.width;
+        } else {
+            newLeft -= popover.offsetWidth / 2;
+        }
+
+        this.topPosition = this.keyPosition.top + this.keyPosition.height + 7 + window.scrollY;
```

Comment:
This is a distance that we add so, that we have a little bit a space between the key and popover.